### ISSUE TITLE
I had play this game since I was a child。I like this game very much, I want to know do you have the full game codes.I fund something miss and the version may not be latest

### ISSUE THREAD

**Issue by 13728989948** [benjcooley/Revenant#1]

**Comment by lurkorrion** [comment]
Yes, I also think it is not release game code. The game is wonderfull, I'm just stuck in undergrounds (buggy levers like in Gothic I). It would be very nice to get full version of source code. I'm not skillfull programmer but someone could remaster the Game Code and give second life for such impressive product. The game sold on the GOG, but don't properly work without kick and it have restricted low screen resolution.

**Comment by Marisa-Chan** [comment]
Of course it's not release code. Seems it's version 0.5 and exe just 1.1mb and in latest release 1.22 it's already 2.0mb. 
Also mostly "clean" for build is walkcode dir and most painfull thing for porting it is many _asm blocks.

**Comment by ghost** [comment]
This git version is mix of prototype, debug and release files in one ...

**Comment by benjcooley** [comment]
Check the issues above for my response.

I've started benjcooley/RevenantRevisited to restore a working version of the game engine and code. And r/RevenantRevisited subreddit to discuss.  I can also potentially dial up a discord if anyone wants. 

Goal is to completely restore the engine and modernize it.  Add new features like moving a variety of bits that should have always been 3D to 3D (like water shader, shadows, etc).
